src/protocolgame.cpp

```diff
@@ -517,19 +517,31 @@ void ProtocolGame::GetTileDescription(const Tile* tile, NetworkMessage& msg)
 		for (auto it = items->getBeginTopItem(), end = items->getEndTopItem(); it != end; ++it) {
 			msg.addItem(*it);
 
-			if (++count == 10) {
+			count++;
```

Comment:
I cry in bed when I see `count++` instead of `++count`

Reply:
Why do you cry?